src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -3211,7 +3082,6 @@ private void handleGetPoLines(RoutingContext ctx) {
         }
       }
     }
-
     private PoLineCollection buildPoLineCollection(String tenant, JsonArray lines) {
```

Comment:
This method needs to be corrected as the objects will not be stored with an id anymore

Reply:
I have removed `replaceObjectById` as all subObjects are part of poLine now.